roadmap.md

```diff
@@ -0,0 +1,81 @@
+# Tekton Mission and Roadmap
+
+This doc describes Tekton's mission and the 2020 roadmap primarily
+for Tekton Pipelines but some sneak peeks at other projects as well.
+
+- [Mission and Vision](#mission-and-vision)
+- [GA releases](#ga-releases)
+- [Release and dogfooding](#release-and-dogfooding)
+- [Results](#results)
+- Tekton Projects
+  - [Pipeline](https://github.com/tektoncd/pipeline/blob/master/roadmap.md)
+  - [Triggers](https://github.com/tektoncd/triggers/blob/master/roadmap.md)
+  - [Catalog](https://github.com/tektoncd/catalog/blob/master/roadmap.md)
+  - Dashboard
+  - CLI
+
+## Mission and Vision
+
+Tekton's mission:
+
+  Be the industry-standard, cloud-native CI/CD platform components and ecosystem.
+
+The vision for this is:
+
+* Tekton API conformance across as many CI/CD platforms as possible
+* A rich catalog of high quality, reusable `Tasks` which work with Tekton conformant systems
+
+What this vision looks like differs across different users:
+
+* **Engineers building CI/CD systems**: These users will be motivated to use Tekton
+  and integrate it into the CI/CD systems they are using because building on top
+  of Tekton means they don't have to re-invent the wheel and out of the box they get
+  scalable, serverless cloud native execution
+* **Engineers who need CI/CD**: (aka all software engineers!) These users will benefit
+  from the rich high quality catalog of reusable components:
+
+  * Quickly build and interact with sophisticated `Pipelines`
+  * Be able to port `Pipelines` to any Tekton conformant system
+  * Be able to use multiple Tekton conformant systems instead of being locked into one
+    or being forced to build glue between multiple completely different systems
+  * Use an ecosystem of tools that know how to interact with Tekton components, e.g.
+    IDE integrations, linting, CLIs, security and policy systems
+
+## GA releases
+
+In 2019 we got to the point where we had several projects built on top of Tekton
+that were craving additional API stability, so we started a push toward having
+a beta release of Tekton. We have
+[created a plan](https://docs.google.com/document/d/1H8I2Rk4kLdQaR4mV0A71Qbk-1FxXFrmvisEAjLKT6H0/edit)
+which defines what Beta means for Tekton Pipelines, and
+[through our beta working group](https://github.com/tektoncd/community/blob/master/working-groups.md#beta-release)
+we are working towared a beta release in early 2020.
+
+After our initial Tekton Pipelines beta release, which
+[does not include all Tekton Pipeline components](https://docs.google.com/document/d/1H8I2Rk4kLdQaR4mV0A71Qbk-1FxXFrmvisEAjLKT6H0/edit#heading=h.t0sc4hdrr5yq),
+we will work toward:
+
+1. Beta and GA for all Tekton Projects
```

Comment:
Should we say *all* or the *core* component (and listing thoses: pipeline, triggers, cli, dashboard, ?)
An example is the new hub project, might not follow the exact same pattern (or any newly added project)

Reply:
kk sgtm!